Library/Homebrew/cask/audit.rb

```diff
@@ -476,7 +537,6 @@ def check_bitbucket_repository
 
     def get_repo_data(regex)
       return unless online?
-      return unless new_cask?
```

Comment:
Would be nice to use this function everywhere instead of `@new_cask`

Reply:
I don't understand? This is a removal, so this would allow for that

Reply:
Yeh, sorry, I'm pointing out that elsewhere in this file it uses `@new_cask` when I think it'd be nicer to use `new_cask?`

Reply:
I'm not sure what the difference is, but sure